piton/design/chipset/rtl/chipset_impl.v.pyv

```diff
@@ -946,8 +946,8 @@ uart_top        uart_top (
 `endif // PITONSYS_NO_MC
 
     // Uncomment to connect to the switch
-    // .uart_timeout_en(uart_timeout_en),
-    .uart_timeout_en            ( 1'b1                                  ),
+    .uart_timeout_en(uart_timeout_en),
+    // .uart_timeout_en            ( 1'b1                                  ),
```

Comment:
What's the original for this? Would be good if this weren't changed versus upstream